static/css/restyle/restyle.less

```diff
@@ -1822,3 +1822,231 @@ h3.author .transfer-ownership {
   margin-left: 0;
   width: 77%;
 }
+
+/* Editor pages */
+body.editor-tools {
+
+  .site-title strong {
+    color: @link-on-color-bg;
+  }
+
+  .pill-nav-amo {
+    border-radius: 0;
+    background-color: transparent;
+    background-image: none;
+  }
+
+  .pill-nav-amo a {
+    color: @link-on-color-bg;
+    border-radius: 0;
+  }
+
+  .pill-nav-amo a.selected {
+    background-color: #ffffff;
+    color: @link-on-white-bg;
+    background-image: none;
+  }
+
+  #navbar {
+    border: none;
+    overflow: visible;
+  }
+
+  #navbar ul li.top {
+    width: 20%;
+    border-right: 0;
+    text-align: center;
+  }
+
+  #navbar > ul > li > ul {
+    background-color: #fff;
+    border: none;
+    border-radius: 0 !important;
```

Comment:
Probably not. I tried to avoid using `!important` as much as possible. Sometimes, there was just no other way.